### DISCUSSION TITLE
Update template's Material library to Material 3

### DISCUSSION THREAD

**Discussion by kaungkhantsoe** [nimblehq/android-templates#594] [RFCs]
## Issue
 Our template is still using old material library

## Solution
- We should update to use the latest Material 3 library to properly manage the colors and fonts
- We will also refactor some of the codes inside the template to work with material 3 library

## Who benefits?
Android developers

## What's Next?

**Comment by hoangnguyen92dn** [comment]
@kaungkhantsoe How about not just Material library? Upgrading BOM and other dependencies also helpful and provide the latest updates for consumers.

**Reply by kaungkhantsoe** [reply]
@hoangnguyen92dn Sounds good 👍 I will try to update as much as possible.
